exercises/grade-school/GradeSchool.elm

```diff
@@ -12,7 +12,7 @@ type alias Student =
 
 
 type alias School =
-    Dict Int (List Student)
+    Dict Grade (List Student)
```

Comment:
Is this supposed to show up on this PR? It's kind of out-of-scope of Bob test updates.

Reply:
yes sorry I forgot to rebase my branch

Reply:
since the other PR's been merged, let's not revert that commit

Reply:
You should be able to just rebase onto master and delete that commit with
```
$ git rebase -i master
```
GitHub might not remove the commit from the UI (they cache the commits), but it'll be gone as a change.

Reply:
true I'd forgotten about that.